src/main/java/org/molgenis/MySpringBootRouter.java

```diff
@@ -102,17 +114,17 @@ public void configure() {
     from("direct:map-alissa-result")
         .split().body()
         .process().body(Map.class, alissaTableMapper::mapLine)
-        .to(marshalVkglResults);
+        .to("direct:marshal-vkgl-result");
```

Comment:
I know sonar isn't happy with repeated strings and in general I agree but for route names I think it actually is clearer than extracting constants.
Bit like in test cases, where repeating strings is also a good thing.

Reply:
Agree. I just wanted to please the sonar gods. Should we silence them?

Reply:
Make them go away! :)